cpp/src/arrow/array-test.cc

```diff
@@ -33,15 +33,12 @@ namespace arrow {
 
 class TestArray : public ::testing::Test {
  public:
-  void SetUp() {
-    pool_ = default_memory_pool();
-  }
+  void SetUp() { pool_ = default_memory_pool(); }
```

Comment:
It is.  The current configuration is to only do one liners for methods declared inside a class scope.    The options are never have single line methods, only have single line methods for empty methods, class-scope or always make single line methods.   We could likely work around this particular instance by adding a line comment to prevent the collapse.

My general preference is for never or class scope.  There were a few instances where the class-scope inlining was already done, and looked right to me.